tests/modules/nf-core/kmcp/compute/test.yml

```diff
@@ -0,0 +1,23 @@
+- name: kmcp compute test_kmcp_compute
+  command: nextflow run ./tests/modules/nf-core/kmcp/compute -entry test_kmcp_compute -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/kmcp/compute/nextflow.config
+  tags:
+    - kmcp/compute
+    - kmcp
+  files:
+    - path: output/kmcp/tmp/_info.txt
```

Comment:
Is there no nice string in there *e.g. 'completed') or something to indicate it ran correctly?